app/controllers/games_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Comment necessary for Rubocop
+class GamesController < ApplicationController
+  before_action :set_game, only: %i[show update destroy]
+
+  # GET /games
+  def index
+    @games = Game.all
+
+    render json: @games
+  end
+
+  # GET /games/1
+  def show
+    render json: @game
+  end
+
+  # POST /games
+  def create
+    Game.delete_all
+    new_game = TicTacToeLogic::GameBoardData.new
+    @game = Game.new({ "game_state": "IN-PROGRESS", "current_player": new_game.current_player.to_s, "board": new_game })
+
+    if @game.save
+      render json: @game, status: :created, location: @game
```

Comment:
```suggestion
      render json: @game, status: :created, location: @game, game_state: @game.game_state, current_player: @game.current_player
```

Reply:
not that 

@ad.attributes.merge

https://stackoverflow.com/questions/33917093/how-to-merge-another-field-in-object-in-rails-json-response/33917257